app/javascript/mastodon/locales/fr.json

```diff
@@ -180,7 +180,7 @@
   "status.show_less": "Replier",
   "status.show_more": "Déplier",
   "status.unmute_conversation": "Ne plus masquer la conversation",
-  "status.unpin": "Unpin from profile",
+  "status.unpin": "Désépingler du profil",
```

Comment:
I understand why, but I don't think `désépingler` is a good idea. Why not "relâcher" like Firefox does?

Reply:
I think "_Relâcher_" is too vague. "_Désépingler_" is a valid word _and_ more precise so why not~

Reply:
I'm not too fond of "désépingler", which sounds like it was made up for the occasion -- although it does exist in French as a construction based on "épingler", its meaning is "remove pins from sth" rather than "free sth from the the pins which held it".

How about "Retirer"? Its common meaning include "remove a publication from display".

Reply:
I went with "désépingler" because this is how it was translated in French in Twitter's UI 
![image](https://user-images.githubusercontent.com/1409924/29752169-f943a8c0-8b0d-11e7-84ef-a60d0ffddfc5.png)

As much as the word seems "weird" I agree with @Komic that it's more precise anyway, and will feel familiar instantly to those who used Twitter before coming to Mastodon since it's basically the same concept/feature.

Reply:
I'm not too fond of a translation choice that would be summed up as "let's use twitter-specific unusual French rather than plain simple French". In translation as in many other domains, simplicity is important.